src/components/categories/categories-card.tsx

```diff
@@ -0,0 +1,29 @@
+import Image from "next/image";
+
+export interface CategoriesCardProps {
```

Comment:
There's no need to export this type.
Also, I suggest to use `type` over `interface`

This relates to props of other components too